test/integration_test/testspec/migration/simple-daemonset.yaml

```diff
@@ -124,13 +124,13 @@ spec:
             httpGet:
               host: 127.0.0.1
               path: /status
-              port: 9001
+              port: 9002
```

Comment:
why do we need to change the port?

Reply:
seems we changed the start port, but forgot to shift the healthy check port, I found this when comparing the pod specs.